Invoke-Parallel/Invoke-Parallel.ps1

```diff
@@ -175,9 +175,10 @@
 			[switch]$NoCloseOnTimeout = $false,
 
             [int]$MaxQueue,
-
+        
+        [Parameter(Mandatory=$false)]
```

Comment:
Leaving this for now, just an fyi - this is the case by default, no need to specify mandatory=$false

Cheers!